test/libsolidity/JSONCompiler.cpp

```diff
@@ -92,7 +92,7 @@ BOOST_AUTO_TEST_CASE(basic_compilation)
 	BOOST_CHECK(dev::jsonCompactPrint(result["sources"]["fileA"]["AST"]) ==
                 "{\"attributes\":{\"absolutePath\":\"fileA\",\"exportedSymbols\":{\"A\":[1]}},"
                 "\"children\":[{\"attributes\":{\"baseContracts\":[null],\"contractDependencies\":[null],"
-                "\"contractKind\":\"contract\",\"fullyImplemented\":true,\"linearizedBaseContracts\":[1],"
+		"\"contractKind\":\"contract\",\"documentation\":null,\"fullyImplemented\":true,\"linearizedBaseContracts\":[1],"
```

Comment:
The new indentation seems to follow the coding standard, but the line stands out.  Maybe in a separate commit, you can fix the indentation around here.